src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write your code here
+const population = document.querySelectorAll('.population');
+
+const content = Array.from(population).map(element => element.textContent);
+
+const contentNew = [];
+
+for (let i = 0; i < content.length; i++) {
```

Comment:
Instead of using a for loop to create a new array, you can try to use `map` method.

`const contentNew = content.map(item => Number(item.replace(/,/g, '')))`